index.html

```diff
@@ -1,46 +1,44 @@
-			<div class="banner">
-				<img src="assets/banner.webp" draggable="false" alt="Profile Banner">
-			</div>
-			<div class="fullpfp">
-				<div class="pfpbg">
-					<div class="pfp">
-						<img src="assets/profil.webp" draggable="false" alt="Profile Picture">
-					</div>
-				</div>
+
+<head>
+	<title>lennartloesche.de</title>
+	<link rel="stylesheet" href="assets/style.css" as="style" />
+	<link rel="shortcut icon" type="image/ico" href="favicon.ico" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css" />
+</head>
+
+<body>
+	<div class="profile m-auto">
+		<div class="banner">
+			<img src="assets/banner.webp" draggable="false" alt="Profile Banner" />
+		</div>
+		<div class="pfpbg ms-4">
+			<img src="assets/profil.png" class="w-100 rounded-circle" draggable="false" alt="Lennart Profile Picture" />
+		</div>
+		<div id="container">
+			<div class="test ms-4">
+				<span class="h4 text-white whitney">Lennart</span><span class="h4 text-white-50 whitney">#0001</span>
 			</div>
-			<div class="pbg">
-				<div class="dcfullname">
-					<div id="container">
-						<div class="test"><span class="name whitney">Lennart</span><span class="discrim whitney">#0001</span></div>
-						<div class="badges">
-							<a href="https://github.com/l3nnartt" class="badges2"><img draggable="false" src="assets/badges/github.png" alt="GitHub Logo"></a>
-							<a href="https://twitter.com/l3nnartt" class="badges2"><img draggable="false" src="assets/badges/twitter.png" alt="Twitter Logo"></a>
-							<a href="https://open.spotify.com/user/g4fa1a33h626g3ztl23p0gys6" class="badges2"><img draggable="false" src="assets/badges/spotify.png" alt="Spotify Logo"></a>
-						</div>
-					</div>
-				</div>
-				<hr class="hr">
-				<div class="aboutme one">
-					<div class="amheader">
-						<p><b>ABOUT ME</b></p>
-					</div>
-					<p>Hey there! My name is Lennart! <br>I'm a still-learning developer from Germany, actively interested in evolving more and more.</p>
-					<p style="font-size: 17px;">
-						<a href="impressum.html">Impressum</a> -
-						<a href="datenschutz.html">Datenschutz</a>
-					</p>
-				</div>
+			<div class="justify-content-center my-auto">
+				<a href="https://github.com/l3nnartt" class="me-3 text-white"><i class="bi bi-github"></i></a>
+				<a href="https://twitter.com/l3nnartt" class="me-3 text-primary"><i class="bi bi-twitter"></i></a>
+				<a href="https://open.spotify.com/user/g4fa1a33h626g3ztl23p0gys6"
+					class="me-3 text-success w-100 me-4"><i class="bi bi-spotify"></i></a>
 			</div>
 		</div>
-	</body>
+		<div class="text-white-50 one mx-4 mt-4 pb-3">
+			<p class="fw-bold">ABOUT ME</p>
```

Comment:
Instead of hard code the whole sentence, use the bootstrap class "text-uppercase". Also useful for SEO.

Reply:
just forgot to implement that